SCAutolib/models/gui.py

```diff
@@ -385,3 +387,24 @@ def assert_no_text(self, key: str, timeout: float = 0):
             if selection.sum() != 0:
                 raise Exception(f"The key='{key}' was found "
                                 f"in the screenshot {screenshot}")
+
+    @log_decorator
+    def check_home_screen(self, *args):
```

Comment:
Could you please add a Docstring to the function?

Reply:
done